client/components/shipping-rates/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import Dropdown from 'components/dropdown';
+
+const ShippingRates = ( { selectedRates, availableRates, packages, updateValue, dimensionUnit, weightUnit, currencySymbol, errors, layout } ) => {
+	const renderTitle = ( pckg ) => {
+		const dimensions = [ pckg.length, pckg.width, pckg.height ].map( dim => dim + ' ' + dimensionUnit ).join( ' x ' );
+		const weight = pckg.weight + ' ' + weightUnit;
+		return <span>{ dimensions + ' ' + weight }</span>;
```

Comment:
The template string is much better.

Just for kicks, here's the sprintf version: ` sprintf( '%1$s %4$s x %2$s %4$s x %3$s %4$s %5$s %6$s ', pckg.length, pckg.width, pckg.height, dimensionUnit, pckg.weight, weightUnit )`

I think we'll need a tie breaker - you up for it @allendav? :P